admin/tests/integration/components/certification-center-form-test.js

```diff
@@ -0,0 +1,40 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import EmberObject from '@ember/object';
+import { selectChoose } from 'ember-power-select/test-support/helpers';
+
+module('Integration | Component | certification-center-form', function(hooks) {
+
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.onSubmit = () => {};
+    this.onCancel = () => {};
+    this.certificationCenter = EmberObject.create();
+  });
+
+  test('it renders', async function(assert) {
+    // when
+    await render(hbs`<CertificationCenterForm @certificationCenter={{this.certificationCenter}} @onSubmit={{action onSubmit}} @onCancel={{action onCancel}} />`);
```

Comment:
`await render(hbs`<CertificationCenterForm @certificationCenter={{this.certificationCenter}} @onSubmit={{this.onSubmit}} @onCancel={{this.onCancel}} />`);`

Reply:
Done